build.gradle

```
@@ -59,6 +43,38 @@ plugins {
   id 'checkstyle'
 }
 
+configurations.all {
+  resolutionStrategy {
+    force("org.apache.httpcomponents.client5:httpclient5:5.4.4")
```

See if you can fetch the version from the catalog
```suggestion
    force("org.apache.httpcomponents.client5:httpclient5:${versions.httpclient5}")
```

---

I tried fetching from catalog but failed